CHANGELOG.md

```diff
@@ -1,6 +1,86 @@
+* Optimize the point select performance and improve the point select efficiency result of Sysbench by 60% 
+* Improve the performance of TiDB on inserting or updating wide tables by 20 times
+* Support configuring the memory upper limit of a single statement in the configuration file
+* Optimize the execution of Hash Join, if the Join type is Inner Join or Semi Join and the inner table is empty, return the result without reading data from the outer table
+* Support using the [`EXPLAIN ANALYZE` statement](https://www.pingcap.com/docs/sql/understanding-the-query-execution-plan/#explain-analyze-output-format) to check the runtime statistics including the execution time and the number of returned rows of each operator
+
+### Statistics
+* Support enabling auto ANALYZE statistics only during certain period of the day
+* Support updating the table statistics automatically according to the feedback of the queries
+* Support configuring the number of buckets in the histogram using the `ANALYZE TABLE WITH BUCKETS` statement
+* Optimize the Row Count estimation algorithm using histogram for mixed queries of equality query and range queries
+
+### Expressions
+* Support following built-in function:
+    - `json_contains`  
+    - `json_contains_path` 
+    - `encode/decode` 
+
+### Server
+* Support queuing the locally conflicted transactions within tidb-server instance to optimize the performance of conflicted transactions
+* Support Server Side Cursor
+* Add the [HTTP API](https://github.com/pingcap/tidb/blob/master/docs/tidb_http_api.md)
+    - Scatter the distribution of table Regions in the TiKV cluster
+    - Control whether to open the `general log`
+    - Support modifying the log level online
+    - Check the TiDB cluster information
+* [Add the `auto_analyze_ratio` system variables to contorl the ratio of Analyze](https://pingcap.com/docs/FAQ/#whats-the-trigger-strategy-for-auto-analyze-in-tidb)
+* [Add the `tidb_retry_limit` system variable to control the automatic retry times of transactions](https://pingcap.com/docs/sql/tidb-specific/#tidb-retry-limit)
+* [Add the `tidb_disable_txn_auto_retry` system variable to control whether the transaction retries automatically](https://pingcap.com/docs/sql/tidb-specific/#tidb-disable-txn-auto-retry)
+* [Support using`admin show slow` statement to obtain the slow queries ](https://pingcap.com/docs/sql/slow-query/#admin-show-slow-command)
+* [Add the `tidb_slow_log_threshold` environment variable to set the threshold of slow log automatically](https://pingcap.com/docs/sql/tidb-specific/#tidb_slow_log_threshold) 
+* [Add the `tidb_query_log_max_len` environment variable to set the length of the SQL statement to be truncated in the log dynamically](https://pingcap.com/docs/sql/tidb-specific/#tidb_query_log_max_len)
+
+### DDL
+* Support the parallel execution of the Add index statement and other statements to avoid the time consuming Add index operation blocking other operations
```

Comment:
```suggestion
* Support the parallel execution of the add index statement and other statements to avoid the time consuming add index operation blocking other operations
```